### ISSUE TITLE
Remove kgio to allow Windows dev environments

### ISSUE THREAD

**Issue by Roenok** [girldevelopit/gdi-new-site#174]
**Description**
The kgio gem is part of the initial bundle install. However, there is no Windows version of kgio, preventing Windows users from setting up a dev environment. 

**Ask**
Figure out an alternative so Windows users can set up a dev environment

**To Do**
- [x] Determine which package is requiring kgio
- [x] Determine is kgio is required for site function
- [x] If kgio is not required, exclude from bundle somehow
- [x] Ig kgio is required, look into alternatives

**Additional Info**
None

**Comment by juliaelman** [comment]
@Roenok would you mind re-formatting using the following template in #83?

**Comment by juliaelman** [comment]
AFAIK kgio is not required for development and can be written in as a condition in the gem file if you are running on a Windows platform. 

@kstack7 thoughts on how to move forward?

**Comment by juliaelman** [comment]
@Roenok and I determined that this issue had to do with running the `unicorn` gem on Windows. We were able move forward, albeit with a new error message, once we commented our that gem from the Gemfile.

@kstack7 since we ran into other issues once the was commented out, should we move forward with making this solve or nix it?

**Comment by kstack7** [comment]
eh hemmmm

![1018152354a](https://cloud.githubusercontent.com/assets/6612770/10569441/0af4ffdc-75f4-11e5-9382-5f1a17c74097.jpg)

So! I'll be cleaning up and double-checking the changes I made to get the site up on a Windows platform. The major changes consist of adding platform checks on specific gems e.g. 'unicorn' so that it will not be included in Windows bundles alongside some other Windows-specific needs.

I had to bump Ruby to 2.1.7 for this to work since 2.1.2 isn't available on Windows. I want to test staging with 2.1.7 so that we could possibly have all platforms using the same version.

ImageMagick is definitely required so the setup process needs to mention it (see #130 for more discussion).

GDI dev work is possible, PC friends!! :D
